docs/containers.conf.5.md

```diff
@@ -10,12 +10,16 @@ and modify the defaults for running containers on the host. containers.conf uses
 a TOML format that can be easily modified and versioned.
 
 Container engines read the /usr/share/containers/containers.conf and
-/etc/containers/containers.conf files if they exists.  When running in rootless
-mode, they also read $HOME/.config/containers/containers.conf files.
+/etc/containers/containers.conf, and /etc/containers/containers.conf.d/*.conf files
+if they exists.  When running in rootless mode, they also read
```

Comment:
```suggestion
if they exist.  When running in rootless mode, they also read
```

Reply:
English sucks as a lanquage.


“Exist” can be either the infinitive, as in “the universe began to exist,” or the present tense for all persons and numbers except third person singular, as in “I exist, you exist, we exist, they exist.” “Exists” is the third person singular present tense: “he exists, she exists, it exists.”